package.json

```diff
@@ -3,14 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.25",
+    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/react-fontawesome": "^0.1.6",
     "@material-ui/core": "^4.5.1",
     "axios": "^0.19.0",
+    "extends": "^0.0.4",
     "formik": "^1.5.8",
+    "less-watch-compiler": "^1.14.1",
```

Comment:
`less-watch-compiler` should be installed globally on your computer.

`yarn global add less-watch-compiler` or
`npm install -g less-watch-compiler`

Reply:
right you are.  not sure why I added it with just yarn add.  nice catch, need to remove as much bloat as we can from libraries